src/components/FirstLaunch/FirstLaunchDialog.module.css

```diff
@@ -0,0 +1,4 @@
+.button_redirect {
+    text-decoration: none !important;
```

Comment:
Hey
I tested on both Mozilla and Chrome and the link styles were applied without the `!important` keyword. Did I overlook any use case or maybe the keyword can be removed?
Thanks!